src/asm/fstack.c

```diff
@@ -133,6 +140,25 @@ void fstk_AddIncludePath(char const *path)
 	includePaths[nbIncPaths++] = str;
 }
 
+void fstk_AddIncludeFilePath(char const *path)
+{
+	struct IncludeFile *file = malloc(sizeof(*file));
+
+	if (!file)
+		fatalerror("Failed to allocate memory for included file: %s\n", strerror(errno));
+	file->path = path;
+	file->next = NULL;
+
+	struct IncludeFile *lastFile = includeFiles;
+
+	while (lastFile && lastFile->next)
+		lastFile = lastFile->next;
+	if (lastFile)
```

Comment:
`(lastFile ? lastFile->next : includeFiles) = file;` would be valid in C++ but not in C.

Reply:
Apparently `*(lastFile ? &lastFile->next : &includeFiles) = file;` is valid C; not sure if that's going too far.